pages/guides.md

```diff
@@ -15,23 +15,43 @@ Beyond helping 18F staff do their best work, we hope these guides will inspire o
 These guides, and all of 18F's work, are in the [worldwide public domain](https://github.com/18F/18f.gsa.gov/blob/main/LICENSE.md){:.usa-link--alt}. You are free to copy and adapt them as you choose. If you see something amiss or want to suggest a change, there are links at the bottom of each guide to help you contribute.
 {% endcapture%}
 
-<section class="usa-section usa-section--dark section-padding-sm bg-primary-darker">
+<section class="usa-section section-padding-sm bg-base-lightest">
   <div class="grid-container">
     <div class="grid-row">
       <div class="grid-col-12 font-sans-lg">
         {{ intro | markdownify }}
       </div>
     </div>
     <div class="grid-row grid-gap margin-top-5">
+    
       {% for guide in site.data.guides %}
-        <div class="tablet:grid-col-6 margin-bottom-5">
-          {% include card-with-image.html 
-             text_content=guide.name
-             link_url=guide.link
-             image_path=guide.image.dark
-             image_side="right"
-          %}
-        </div>
+        {% if guide.promoted == true %}
```

Comment:
**suggestion (non-blocking)**: The structure here where we use the same loop twice with disjoint if statements is not completely obvious. Maybe use a comment tag to say something about it like
```suggestion
        {% # loop only over the promoted items first %}
        {% if guide.promoted == true %}
```